server/src/test/java/org/elasticsearch/search/sort/GeoDistanceSortBuilderTests.java

```diff
@@ -46,6 +49,25 @@
 
 public class GeoDistanceSortBuilderTests extends AbstractSortTestCase<GeoDistanceSortBuilder> {
 
+    private static final int NUMBER_OF_TESTBUILDERS = 20;
+
+    @Override
+    public void testFromXContent() throws IOException {
+        for (int runs = 0; runs < NUMBER_OF_TESTBUILDERS; runs++) {
+            GeoDistanceSortBuilder testItem = randomGeoDistanceSortBuilder();
+            XContentBuilder builder = XContentFactory.contentBuilder(randomFrom(XContentType.values()));
+            testItem.toXContent(builder, ToXContent.EMPTY_PARAMS);
+            XContentBuilder shuffled = shuffleXContent(builder);
+            try (XContentParser parser = createParser(shuffled)) {
+                parser.nextToken();
+                NestedSortBuilder parsedItem = NestedSortBuilder.fromXContent(parser);
```

Comment:
I think this is not right? It does not need to be a nested sort

Reply:
thought I'd fixed that, I must have merged it back in, thanks